code/modules/mob/living/simple_animal/hostile/mining_mobs.dm

```diff
@@ -753,6 +753,11 @@
 	layer = MOB_LAYER-0.1
 	loot = list(/obj/effect/gibspawner)
 	del_on_death = 1
+	var/gps = null
+
+/mob/living/simple_animal/hostile/spawner/lavaland/New()
+	gps = new /obj/item/device/gps/internal(src)
```

Comment:
Probably better to do this after calling `..()` instead of before. Shouldn't make much difference but it's usually safer.